daemon/src/model.rs

```diff
@@ -127,6 +127,15 @@ impl BitMexPriceEventId {
         Self::new(timestamp, 20)
     }
 
+    /// Checks whether this event has likely already occurred.
+    ///
+    /// We can't be sure about it because our local clock might be off from the oracle's clock.
+    pub fn has_likely_occured(&self) -> bool {
+        let now = OffsetDateTime::now_utc();
+
+        self.timestamp > now
```

Comment:
:mega: Isn't this the wrong way round? We want the event's timestamp to be in the past.